react_ui/src/pages/Home.tsx

```diff
@@ -1,17 +1,32 @@
-import React from 'react'
-import { useTranslation } from 'react-i18next'
+import React, { useState } from 'react'
 
+import { classnames } from '../components/classnames'
 import { ContentContainer } from '../components/ContentContainer'
 
 export const Home: React.FC = () => {
-  const { t } = useTranslation()
+  const [isOpen, setOpen] = useState(false)
+  let sideBarDummyContents = ''
+  for (let i = 0; i < 300; i++) sideBarDummyContents += i + ' '
 
   return (
     <ContentContainer>
-      <div className="grid grid-cols-1 gap-4 m-16">
-        <h1 className="font-bold text-3xl mb-6">{t('home.pageTitle')}</h1>
+      <div>
+        <h1 className="font-bold text-3xl m-6">Shipments</h1>
```

Comment:
This page is basically a stub for now. Will use i18n on the actual page when we're ready to implement it.